packages/monaco-graphql/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "monaco-graphql",
+  "description": "full service, official monaco mode for GraphQL",
+  "version": "2.3.4-alpha.5",
```

Comment:
we can decide what this should be before we release a stable version, but I'll probably just revert to 0.0.1 for the first pre-release